tests/CI/exportCSLoop.sh

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 # This script will export to the `Production.cfg` file to the
 # yaml format for diracx every 5 seconds
+set -x
+exec >>/tmp/cs-loop.log 2>&1
 
+while [[ ! -f /home/dirac/ServerInstallDIR/bashrc ]]; do
+    sleep 1;
+done
+sleep 1;
 source /home/dirac/ServerInstallDIR/bashrc
+
 git config --global user.name "DIRAC Server CI"
 git config --global user.email "dirac-server-ci@invalid"
 
-while true;
-do
-    curl -L https://gitlab.cern.ch/chaen/chris-hackaton-cs/-/raw/master/convert-from-legacy.py |DIRAC_COMPAT_ENABLE_CS_CONVERSION=True  ~/ServerInstallDIR/diracos/bin/python - ~/ServerInstallDIR/etc/Production.cfg /cs_store/initialRepo/
-    git -C /cs_store/initialRepo/ commit -am "export $(date)"
+while true; do
+    curl -L https://gitlab.cern.ch/chaen/chris-hackaton-cs/-/raw/integration-tests/convert-from-legacy.py |DIRAC_COMPAT_ENABLE_CS_CONVERSION=True  /home/dirac/ServerInstallDIR/diracos/bin/python - /home/dirac/ServerInstallDIR/etc/Production.cfg /cs_store/initialRepo/
```

Comment:
I'll try to take care of https://github.com/DIRACGrid/diracx/issues/68 on Monday

Reply:
It was already done in https://github.com/DIRACGrid/diracx/pull/115